test/unit/feature_engineering/test_apply_features.py

```diff
@@ -95,9 +95,31 @@ def test_apply_features(self):
 
         self.assertDataframeEqual(
             input_data_with_features,
-            expected_output.drop(
-                columns=["is_bevrijdingsdag", "is_bridgedaybevrijdingsdag"]
-            ),
+            expected_output,
+            check_like=True,  # ignore the order of index & columns
+            check_less_precise=3,  # ignore small rounding differences
+        )
+
+    def test_apply_features_no_pj(self):
+        """pj = None should work fine as well"""
+        # Arrange
+        input_data_without_features = TestData.load("input_data.pickle")
+        pj = None
+        expected_output = TestData.load("input_data_with_features.csv")
+
+        # Act
+        input_data_with_features = apply_features.apply_features(
+            input_data_without_features,
+            horizon=24,
+            pj=pj,
+        )
+
+        # Assert:
+        assert len(input_data_with_features) == len(input_data_without_features)
+        assert ["gti", "dni"] not in list(input_data_with_features.columns)
```

Comment:
I'm confused what you want if pj=None; gti and dni to be calculated using the default location, or to be not calculated?